crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -19,161 +19,214 @@
 # * Tweet not related to weather condition
 #
 # The catch is that 20 contributors graded each tweet, and in many cases contributors assigned
-# conflicting sentiment labels to the same tweet.
+# conflicting sentiment labels to the same tweet. Our goal is to label each tweet as either
+# positive or negative so that we can train a language model over the tweets themselves.
+#
 # This is a common issue when dealing with crowdsourced labeling workloads.
```

Comment:
This line was meant to follow "contributors assigned conflicting labels...". You can delete or move it.